examples/bigcommerce/README.md

```diff
@@ -31,69 +31,21 @@ BIGCOMMERCE_ACCESS_TOKEN=5lw9ulikcp186tjgg3rs39kh4fg3vci
 
 ---
 
-## Using this repo
+## Using this repo locally
 
-1. **Create a site in Makeswift and a shop in BigCommerce**
-
-   Head over to [Makeswift](https://app.makeswift.com) and sign up for a free account. Create a site using the option to "Integrate with Next.js",
-   and copy the Site API key. You will need it in step 3.
-
-   Then head over to [BigCommerce](https://www.bigcommerce.com/) and create a store. Here is [a guide](https://support.bigcommerce.com/s/article/Starting-a-BigCommerce-Trial) to get you started.
-
-   Note: If you are just trying out Makeswift with our e-commerce template feel free use this example store and the read-only access token for your development.
-
-   ```
-   BIGCOMMERCE_STORE_NAME=makeswift-example
-   BIGCOMMERCE_STORE_HASH=uvhswop3wh
-   BIGCOMMERCE_ACCESS_TOKEN=5lw9ulikcp186tjgg3rs39kh4fg3vci
-   ```
-
-2. **Clone this template**
-
-   Instead of using `create-next-app`, run this command from the terminal:
-
-   ```bash
-   npx degit makeswift/makeswift/examples/bigcommerce bigcommerce
-   cd bigcommerce
-   ```
-
-   It will download this subdirectory of the `makeswift/makeswift` repo without including git history.
-
-3. **Update environment variables**
-
-   Rename `.env.local.example` to `.env.local` and update it with values from your BigCommerce and Makeswift accounts.
-
-   - `BIGCOMMERCE_STORE_NAME` and `BIGCOMMERCE_STORE_HASH` can be found in the BigCommerce dashboard
-
-       <img src="https://user-images.githubusercontent.com/20950876/184250701-c7af5854-ad4a-4dec-b8e1-1653cddbff1c.png" width="600" />
-
-   - `BIGCOMMERCE_ACCESS_TOKEN` requires you to [create an API account](https://support.bigcommerce.com/s/article/Store-API-Accounts?language=en_US)
```

Comment:
It isn't clear after the change where they would get the `BIGCOMMERCE_ACCESS_TOKEN`